### ISSUE TITLE
Questions about g_path_regularize function

### ISSUE THREAD

**Issue by chengfeng0113** [rosinality/stylegan2-pytorch#32]
Hi,
   Thank you so much for sharing this, I am also studying how to implement stylegan2 using pytorch. At present, the overall framework is basically completed, but there are still some details to be implemented. On how to implement delayed regularization with a single machine and multiple graphics cards.  I am a little confused here about ` path[0].backward()`，In my understanding, path[0]'s grad_fn is False and cannot be back-propagated. Because the grad_fn of the gradient grad is also False in the function g_path_regularize, it is not clear why path_penalty can calculate the gradient by backpropagation alone. Hope to get answers, thank you.

**Comment by rosinality** [comment]
Is grad_fn of path[0] is None? I will re-check it but last time I checked it gradient is computed.

**Comment by chengfeng0113** [comment]
You are right. In function torch.autograd.grad, I did not turn on create_graph = true, resulting in no calculation of higher-order differential, so it cannot be back propagation.
